Classes/Issues/Comments/Reactions/IssueCommentReactionCell.swift

```diff
@@ -153,26 +153,22 @@ UICollectionViewDelegateFlowLayout {
     func configure(cell: IssueReactionCell, model: ReactionViewModel) {
         let users = model.users
         let detailText: String
-        switch users.count {
+        switch model.count {
         case 0:
             detailText = ""
         case 1:
             let format = NSLocalizedString("%@", comment: "")
-            detailText = String.localizedStringWithFormat(format, users[0])
-            break
+            detailText = .localizedStringWithFormat(format, users[0])
```

Comment:
Why is this actually not `localizedString(format:)`. Just realizing this now, lol

Reply:
🤷‍♂️